s4e-web/src/app/views/login/login.component.html

```diff
@@ -25,21 +25,21 @@ <h1>
     <div class="content__formContainer">
       <form (ngSubmit)="login()" [formGroup]="form" autocomplete="off">
         <h2 class="content__header" i18n>Logowanie</h2>
-        <div class="alert alert-danger special__error" *ngIf="error$ | async">
+        <div *ngIf="error$ | async" class="alert alert-danger special__error">
           <ng-container [ngSwitch]="(error$ | async).status">
             <ng-container *ngSwitchCase="401">Niepoprawny login / hasło</ng-container>
             <ng-container *ngSwitchDefault>Nieoczekiwany błąd</ng-container>
           </ng-container>
           <br>
         </div>
 
-        <div class="alert alert-danger special__error" *ngIf="form.errors">
-          <ul>
-            <li *ngFor="let error of (form.errors | errorKeys)" [ngSwitch]="error">
-              <ng-container *ngSwitchDefault i18n>Nieoczekiwany błąd</ng-container>
-            </li>
-          </ul>
-        </div>
+<!--        <div class="alert alert-danger special__error" *ngIf="!(form.errors | isEmpty)">-->
+<!--          <ul>-->
+<!--            <li *ngFor="let error of (form.errors | errorKeys)" [ngSwitch]="error">-->
+<!--              <ng-container *ngSwitchDefault i18n>Nieoczekiwany błąd</ng-container>-->
+<!--            </li>-->
+<!--          </ul>-->
+<!--        </div>-->
```

Comment:
Why have you commented on those lines? Why haven't you just removed them?

Reply:
I removed them because they (as far as I know) didn't work correctly. But this will need to be consulted with somone else, so I just commented it out.